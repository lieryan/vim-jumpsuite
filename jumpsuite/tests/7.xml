<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
	<testsuite errors="1" failures="0" name="comments.tests.test_api.APITests-20190318172159" skipped="0" tests="1" time="0.168" timestamp="2019-03-18T17:22:00">
		<testcase classname="comments.tests.test_api.APITests" name="test_api_by_foo" time="0.168" timestamp="2019-03-18T17:22:00">
			<error message="division by zero" type="ZeroDivisionError">
<![CDATA[Traceback (most recent call last):
  File "/home/user/Projects/myapp/comments/tests/test_api.py", line 130, in test_api_by_foo
    1/0
ZeroDivisionError: division by zero
]]>			</error>
		</testcase>
		<system-out>
<![CDATA[]]>		</system-out>
		<system-err>
<![CDATA[]]>		</system-err>
	</testsuite>
</testsuites>
