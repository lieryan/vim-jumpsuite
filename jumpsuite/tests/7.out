FAILED. Ran 1 tests in 0.168s (errors=1, failures=0, skipped=0)
/home/user/Projects/myapp/comments/tests/test_api.py:130: test_api_by_foo : ZeroDivisionError: division by zero : 1/0
